filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobPathBuilderSpec.scala

```diff
@@ -62,4 +62,15 @@ class BlobPathBuilderSpec extends AnyFlatSpec with Matchers{
     val fileText = (is.readAllBytes.map(_.toChar)).mkString
     fileText should include ("This is my test file!!!! Did it work?")
   }
+
+  ignore should "build duplicate blob paths in the same filesystem" in {
+    val endpoint = BlobPathBuilderSpec.buildEndpoint("coaexternalstorage")
+    val store = "inputs"
+    val evalPath = "/test/inputFile.txt"
+    val sas = "{SAS TOKEN HERE}"
+    val testString = endpoint + "/" + store + evalPath
+    val blobPath1: BlobPath = new BlobPathBuilder(new AzureSasCredential(sas), store, endpoint) build testString getOrElse fail()
+    val blobPath2: BlobPath = new BlobPathBuilder(new AzureSasCredential(sas), store, endpoint) build testString getOrElse fail()
+    blobPath1 should equal(blobPath2)
+  }
```

Comment:
Hmm, this is a good demonstration but I don't know that we actually want to check it in. This is a good topic to discuss in Michael's meeting next week, how should we be handling automated tests for this stuff?

Reply:
Agreed here, I've been leaving this for the time being because I have been using it to verify I haven't broken these changes, but I can remove this before merge